### PR TITLE
React components

### PR DESCRIPTION
**Pull request by brahimdidi**
In this part, I will use React components Classes (Stateful functions) to render the calculator using JSX.
## Requirements 
Delete all the boilerplate from CRA (text, images, styles).
Create a directory called components.
Inside components create a new Calculator.js file.
In Calculator.js, create a React component that matches the design given.
Don't add any extra design.
Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
Use class-based components (All components should be class-based).
Once the Calculator is ready, import it and use it in your main component (App.js).
Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
```
Comment (SajjadAhmad14) [inline src/components/Calculator.js:1]: There is a compilation error. Kindly resolve it and make sure the app is working without any error
errors
.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Didi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Calculator.js (hunk 1 above)
Comment (brahimdidi) [reply src/components/Calculator.js:1]: It does not appear on my part, when you run npm start everything works perfect

Comment (brahimdidi) [reply src/components/Calculator.js:1]: Please check it again it is working properly I tried it many times

**Review by medaminedev66 — APPROVED** [review]
## Hi @brahimdidi, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 

# Highlights
- The Calculator component works as expected
- The calculator design matches the provided design

Keep it up! 😃


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**